webrender/src/prim_store.rs

```diff
@@ -161,6 +166,21 @@ impl<F, T> SpaceMapper<F, T> where F: fmt::Debug {
         }
     }
 
+    pub fn unmap(&self, rect: &TypedRect<f32, T>) -> Option<TypedRect<f32, F>> {
```

Comment:
I think this should be renamed. It sounds like the mapping was reversible in the first place and we just undo it, but in reality `inverse_rect_footprint` is clearly irreversible.